doc/source/actors.rst

```diff
@@ -135,117 +244,205 @@ sufficient CPU resources and the relevant custom resources.
 Terminating Actors
 ------------------
 
-Actor processes will be terminated automatically when the initial actor handle
-goes out of scope in Python. If we create an actor with ``actor_handle =
-Counter.remote()``, then when ``actor_handle`` goes out of scope and is
-destructed, the actor process will be terminated. Note that this only applies to
-the original actor handle created for the actor and not to subsequent actor
-handles created by passing the actor handle to other tasks.
-
-If necessary, you can manually terminate an actor by calling
-``ray.actor.exit_actor()`` from within one of the actor methods. This will kill
-the actor process and release resources associated/assigned to the actor. This
-approach should generally not be necessary as actors are automatically garbage
-collected. The ``ObjectRef`` resulting from the task can be waited on to wait
-for the actor to exit (calling ``ray.get()`` on it will raise a ``RayActorError``).
-Note that this method of termination will wait until any previously submitted
-tasks finish executing and then exit the process gracefully with sys.exit. If you
-want to terminate an actor forcefully, you can call ``ray.kill(actor_handle)``.
+.. tabs::
```

Comment:
Can you add some sub titles in this section? E.e. "Automatic Actor GC", "Terminating an actor within an actor method", "Terminating an actor with the actor handle".